locales-pending/dashboard-premium.ftl

```diff
@@ -189,3 +189,15 @@ exposure-card-description-info-for-sale-action-needed-dashboard = This site is p
 exposure-card-description-info-for-sale-in-progress = We’ve started our auto-removal process of <data_broker_profile>this profile</data_broker_profile> to protect your information. <removal_info>Removals typically take 7-14 days</removal_info>.
 exposure-card-description-info-for-sale-fixed = As a { -brand-monitor-plus } member, we’ve <data_broker_profile>removed this profile</data_broker_profile> for you and will continually monitor to make sure they don’t add you back.
 exposure-card-description-info-for-sale-fixed-manually-fixed =  You marked this profile as fixed. Be sure you’ve followed all instructions on <data_broker_profile>the site</data_broker_profile> to ensure they remove your personal info.
+
+
+# CSAT survey banner for subscribers
```

Comment:
```suggestion
## CSAT survey banner for subscribers
```

Reply:
Resolved in [deeb6b9](https://github.com/mozilla/blurts-server/pull/4421/commits/deeb6b9c10f53ec66b163b59572e22a63436d92f).